action.yml

```diff
@@ -24,6 +29,7 @@ runs:
           --monochrome-output \
           --arg plan "${{ inputs.plan }}" \
```

Comment:
Using single quotes here should do it, no? Singles are for string literals, doubles can contain substitution sequences.

Reply:
oh yeah if its a templating already than that might work.. lets see

Reply:
yep that works
https://github.com/TakeScoop/aws-golden-gate/runs/3500486843?check_suite_focus=true